src/BloomBrowserUI/bookEdit/js/BloomHintBubbles.ts

```diff
@@ -294,33 +294,37 @@ export default class BloomHintBubbles {
 
         // determine onFocusOnly
         var onFocusOnly = whatToSay.startsWith('*');
-        onFocusOnly = onFocusOnly || source.hasClass('bloom-showOnlyWhenTargetHasFocus') || bloomQtipUtils.mightCauseHorizontallyOverlappingBubbles(target);
+        // We seem to need a delay to get a reliable result from mightCauseHorizontallyOverlappingBubbles(); see comment there.
+        setTimeout(() => {
 
-        if (whatToSay.startsWith('*')) whatToSay = whatToSay.substr(1);
-        var functionCall = source.data("functiononhintclick");
-        if (functionCall) {
-            if (functionCall === 'bookMetadataEditor' && !BloomHintBubbles.canChangeBookLicense())
-                return;
-            shouldShowAlways = true;
+            onFocusOnly = onFocusOnly || source.hasClass('bloom-showOnlyWhenTargetHasFocus') || bloomQtipUtils.mightCauseHorizontallyOverlappingBubbles(target);
 
-            if (functionCall.indexOf('(') > 0)
-                functionCall = 'javascript:' + functionCall + ';';
+            if (whatToSay.startsWith('*')) whatToSay = whatToSay.substr(1);
+            var functionCall = source.data("functiononhintclick");
+            if (functionCall) {
```

Comment:
Github is showing a lot of this PR changes when no changes are visible, as if it's just whitespace (reviewable is down). I though we had processed our files and put extensions in place so that all commits should not generate whitespace differences anymore?